addOns/formhandler/src/main/java/org/zaproxy/zap/extension/formhandler/FormHandlerParam.java

```diff
@@ -84,7 +84,22 @@ public class FormHandlerParam extends VersionedAbstractParam {
                             "(?i)_?locale[-_]?(?:code)?",
                             Constant.getSystemsLocale().toLanguageTag(),
                             true,
-                            true));
+                            true),
+                    new FormHandlerParamField(
+                            "(?i)_?(?:comment|subject|summary)?",
+                            "Zaproxy dolore alias impedit expedita quisquam.",
+                            true,
+                            true),
+                    new FormHandlerParamField(
+                            "(?i)_?(?:description|message|(?:email)?[-_]?content)?",
```

Comment:
Reassess brackets on email. Consider adding “post”

Reply:
The brackets seem to be correct and adding post is simply another or condition along with email.

https://www.regexplanet.com/share/index.html?share=yyyypqj1ekr